### DISCUSSION TITLE
can you added dual PCI e one for SSD , 	Intel 8260NGW 802.11ac WiFi M.2 Adapter

### DISCUSSION THREAD

**Discussion by ibrahelsheikh** [geerlingguy/raspberry-pi-pcie-devices#508] [General]

**Comment by geerlingguy** [comment]
Not sure what this issue is requesting, but it should probably be a discussion instead; converting it now.

**Comment by ibrahelsheikh** [comment]
I would like to simultaneously connect an SSD and a WiFi M.2 Adapter.

**Reply by PixlRainbow** [reply]
You will need a PCIE switch that splits out the one channel from the Pi into two (or more) PCIE slots.
Be warned that you will get half the bandwidth on each card; the SSD and the WiFi adapter will not run at full speed.
https://pipci.jeffgeerling.com/#pcie-switches-and-adapters
Take care to note the "functionality" column.

**Reply by PixlRainbow** [reply]
Alternatively, you can choose a carrier board that integrates a PCIE switch/split
https://pipci.jeffgeerling.com/boards_cm
